src/hooks/useMediaQuery/useMediaQuery.stories.mdx

```diff
@@ -4,38 +4,32 @@ import { Meta } from '@storybook/blocks';
 
 # useMediaQuery
 
-Hook that returns a boolean indicating whether the media query matches.
+Hook that returns true when the media query matches.
 
-## Augmenting the MediaQuery type
-
-The `MediaQuery` type is used for the `mediaQueryVariableName` parameter of `useMediaQuery`.
-
-Augment the `MediaQuery` interface to add the names for the CSS variables that describe media
-queries in your project.
-
-```tsx
-declare module '@mediamonks/react-hooks' {
-  interface MediaQuery {
-    '--min-width-480': never;
-    '--min-width-820': never;
-    '--min-width-768': never;
-    '--min-width-1024': never;
-  }
-}
-```
 
 ## Reference
 
 ```ts
-function useMediaQuery(mediaQueryVariableName: keyof MediaQuery, defaultValue = false): boolean;
+function useMediaQuery(mediaQueryOrVariableName: string, defaultValue = false): boolean;
```

Comment:
If we type this as `string`, I think we should more clearly document what types of strings you can pass. Having only the 2 examples further down might not clear enough to understand what's really happening.